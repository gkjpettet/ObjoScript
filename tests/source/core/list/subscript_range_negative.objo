var heroes = ["Iron Man", "Hulk", "Black Panther"]
System.print(heroes[0...-1]) // Expect error