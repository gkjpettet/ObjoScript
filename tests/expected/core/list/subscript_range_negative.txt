Expected range index to be a positive integer. Instead got `-1`.